utils/labs.ts

```
@@ -0,0 +1,51 @@
+        const fileContent = await fs.readFile(labPath, "utf-8");
+
+        // extract metadata from MDX file
+        const metadataMatch = fileContent.match(
+          /export const metadata = ({[\s\S]*?});/,
+        );
+        if (!metadataMatch) {
+          throw new Error(`No metadata found in ${labPath}`);
+        }
+
+        // safely evaluate the metadata object
+        const metadata = eval(`(${metadataMatch[1]})`);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)